### ISSUE TITLE
JLM tests with securirty fail on Windows 

### ISSUE THREAD

**Issue by Mesbah-Alam** [AdoptOpenJDK/openjdk-systemtest#198]
JLM tests with security enabled fail on Windows with : 

```
Successfully processed 1 files; Failed processing 0 files
ERROR: File ownership cannot be applied on insecure file systems;
       there is no support for ACLs.
**FAILED** at step 2 (Re-assign ownership for jmxremote.password file). Expected return value=0 Actual=256 at T://setUp.pl line 73.
STF 19:34:42.184 - **FAILED** setup script failed. Expected return value=0 Actual=1

```

https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/475/tapResults/

**Comment by Mesbah-Alam** [comment]
There is a step in the tests that set the ownership of jmxremote.password file to `600` using `chmod` (an STF api for doing chmod) that doesn't seem to be working on Windows.

**Comment by lumpfish** [comment]
@Mesbah-Alam - I've created a fork of openjdk-tests and updated the playlist there to reinstate the test for testing at AdoptOpenJDK.

**Comment by Mesbah-Alam** [comment]
The test was actually excluded from Windows + OpenJ9. It looks like you've already run another Grinder with it included! https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder-systemtest/170/

**Comment by lumpfish** [comment]
Yes, as I said above, I updated the playlist in my private fork of openjdk-tests.  Unfortumately my openjdk-systemtest branch is now incompatible with the other repositories as a result of merging the (remove top.xml) build changes - I'll need to rebase on the latest master.

**Comment by Mesbah-Alam** [comment]
Yes, please do.
